src/containers/AppAuth0.tsx

```diff
@@ -36,25 +36,13 @@ const AppAuth0: React.FC = () => {
 
   const [, , bankAccountsService] = useMachine(bankAccountsMachine);
 
-  // @ts-ignore
-  if (window.Cypress) {
-    // eslint-disable-next-line react-hooks/rules-of-hooks
-    useEffect(() => {
-      const auth0 = JSON.parse(localStorage.getItem("auth0Cypress")!);
-      authService.send("AUTH0", {
-        user: auth0.user,
-        token: auth0.token,
-      });
-    }, []);
-  } else {
-    // eslint-disable-next-line react-hooks/rules-of-hooks
-    useEffect(() => {
-      (async function waitForToken() {
-        const token = await getAccessTokenSilently();
-        authService.send("AUTH0", { user, token });
-      })();
-    }, [isAuthenticated, user, getAccessTokenSilently]);
-  }
+  // eslint-disable-next-line react-hooks/rules-of-hooks
+  useEffect(() => {
+    (async function waitForToken() {
+      const token = await getAccessTokenSilently();
+      authService.send("AUTH0", { user, token });
```

Comment:
does the auth service still need to be declared on the window above?

Reply:
Yeah it's still used by other tests